### ISSUE TITLE
normalizeSVG does not set width and height properties of exported SVG

### ISSUE THREAD

**Issue by thesuavehog** [amcharts/amcharts4#3156]
**normalizeSVG not setting width/height of SVG**

If you run the 'Generating static charts' example from https://www.amcharts.com/docs/v4/tutorials/generating-static-charts/ and open the Chrome console, you will see copious error messages because the `normalizeSVG` code does not set the width or height attributes on the exported SVG.

Even though the `pixelWidth` & `pixelHeight` are passed into `normalizeSVG`:
```javascript
      var target = chart.chartAndLegendContainer;
      var svg = chart.exporting.normalizeSVG(
        chart.exporting.serializeElement(target.dom),
        {},
        target.pixelWidth,
        target.pixelHeight
      );
```

This happens when I used the libraries as well.

**Environment (if applicable)**

```
<script src="https://www.amcharts.com/lib/4/core.js"></script>
<script src="https://www.amcharts.com/lib/4/charts.js"></script>
<script src="https://www.amcharts.com/lib/4/themes/microchart.js"></script>
```
Chrome on Windows 10

**Comment by martynasma** [comment]
Thanks for the heads up!

We will fix this in the next version. Will let you know when it's out.

In the interim you can easily fix by passing in scale parameter into the `normalizeSVG()`:

```JavaScript
var target = chart.chartAndLegendContainer;
var svg = chart.exporting.normalizeSVG(
        chart.exporting.serializeElement(target.dom),
        {},
        target.pixelWidth,
        target.pixelHeight,
        1
);
```

**Comment by martynasma** [comment]
Fixed in [4.10.13](https://github.com/amcharts/amcharts4/releases/tag/4.10.13).

## [4.10.13] - 2020-12-18

### Added
- Export: PDF export options now support `scale`, `minWidth`, `minHeight`, `maxWidth`, and `maxHeight` options.

### Changed
- Setting `ignoreZeroValues = true` on `PieSeries`/`PyramidSeries` will now hide `null`-value slices, too.

### Fixed
- Inserting a new bullet into an initialized Series will now invalidate it cause the new bullet to appear immediately.
- Using `timezone` might shift axis label values by one day in some cases.
- `"i"` input date format was not properly parsing formats with no milliseconds and with timezone offset, e.g. (`"...+10:30"`).
- Export: `normalizeSVG()` method was producing wrong SVG width/height in output if scale parameter was not being passed in.
- RangeSelector plugin: "YTD" pre-defined period was not working properly with some data-grouping setups.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
